frontend/src/components/user/ATSUserLinkModal.vue

```diff
@@ -9,11 +9,15 @@ import { useAppStore } from '@/store';
 import { Initiative } from '@/utils/enums/application';
 
 import type { Ref } from 'vue';
-import type { ATSClientResource, ElectrificationProject, Enquiry, HousingProject } from '@/types';
+import type { ATSClientResource } from '@/types';
+import { computed } from 'vue';
 
 // Props
-const { projectOrEnquiry } = defineProps<{
-  projectOrEnquiry: Enquiry | HousingProject | ElectrificationProject;
+const { fName, lName, phoneNumber, emailId } = defineProps<{
+  fName: string;
```

Comment:
may as well make prop name `firstName`, not saving anything on abbreviation, same for `lastName`.